client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```
@@ -835,8 +835,14 @@ public final void executeAction(RPCActionDto dto, ActionResponseDto responseDto)
                     responseDto.rpcResponse = resSchema.getDto();
                     if (dto.doGenerateAssertions && dto.responseVariable != null)
                         responseDto.assertionScript = resSchema.newAssertionWithJava(dto.responseVariable, dto.maxAssertionForDataInCollection);
-                    else
-                        responseDto.jsonResponse = objectMapper.writeValueAsString(response);
+                    else{
+                        try {
+                            responseDto.jsonResponse = objectMapper.writeValueAsString(response);
+                        }catch (JsonProcessingException e){
+                            // cannot convert to json
```

so if it happens we silently ignore it?